src/main/java/com/cloudbees/jenkins/plugins/awscredentials/config/AmazonWebServicesCredentialsConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package com.cloudbees.jenkins.plugins.awscredentials.config;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.StaplerRequest;
+
+@Extension
+public final class AmazonWebServicesCredentialsConfiguration extends GlobalConfiguration {
+    private boolean validateAgainstAWS = true;
+
+    public AmazonWebServicesCredentialsConfiguration() {
+        load();
+    }
+
+    public static AmazonWebServicesCredentialsConfiguration get() {
+        return (AmazonWebServicesCredentialsConfiguration) Jenkins.get().getDescriptor(AmazonWebServicesCredentialsConfiguration.class);
+    }
+
+    @Override
+    public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
+        validateAgainstAWS = json.getBoolean("validateAgainstAWS");
+        save();
+        return super.configure(req, json);
```

Comment:
Do you need to override this method? I think the default implementation in the parent class would already do what's expected.

Reply:
From what I see it doesn't. Neither the save nor the setting of the local variable.